docs/tables/aws_securityhub_insight.md

```diff
@@ -0,0 +1,108 @@
+# Table: aws_securityhub_insight
+
+An AWS Security Hub insight is a collection of related findings. It identifies a security area that requires attention and intervention. For example, an insight might point out EC2 instances that are the subject of findings that detect poor security practices. An insight brings together findings from across finding providers.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  arn,
+  group_by_attribute,
+  region
+from
+  aws_securityhub_insight;
+```
+
+### List insights by a particular attribute
+
+```sql
+select
+  name,
+  arn,
+  group_by_attribute,
+  region
+from
+  aws_securityhub_insight
+where
+  group_by_attribute = 'ResourceId';
+```
+
+### Get workflow status details for insights
+
+```sql
+select
+  name,
+  arn,
+  filters ->> 'WorkflowStatus' as workflow_status
+from
+  aws_securityhub_insight;
+```
+
+### Get severity details for insights
+
+```sql
+select
+  name,
+  arn,
+  filters ->> 'SeverityLabel' as severity_label
```

Comment:
I've read more on what insights are and how they're used, and since the insights results are in a separate API (and probably will be in a separate table), this table seems useful for finding insights based on their group by attribute (which has an example already) and filters.

Are there any more useful examples we can add around listing insights based on their filters, e.g., `### List insights that filter on high severity label and ...`? The ones already defined in this doc are useful, so my suggestion is primarily around adding 2-3 more in addition to the current set.

Reply:
@cbruno10 I tried to add example queries related to filtering on the severity label, but couldn't find any scenario where I could get a severity out of the response.
Moreover, the SeverityLabel field is of `[]*StringFilter` which has `Comparison` and `Value` inside it, and it doesn't seem like SeverityLabel.Value is an enum of severity values.

Reply:
Could we include something like this for severity label, and then 1-2 more examples for similar column types? The actual Comparison/Value values can be changed for a better example as well.

```
### List insights that filter on high severity labels 

select
  name,
  insight_arn,
  filters ->> 'SeverityLabel' as severity
from
  aws_securityhub_insight
where
  filters ->> 'SeverityLabel' @> '[{"Comparison": "EQUALS", "Value": "HIGH"}]'
```

For finding severities, I found https://docs.aws.amazon.com/securityhub/latest/userguide/asff-severity.html, which may be the definitive list of what severity labels findings can have. Even if a customer can define a custom value, the example would still be helpful to show users how to interact with columns that have the `[]*StringFilter` type.

Reply:
@cbruno10 Thanks! That was very helpful, added the queries!